File: apps/Application/Models/SystemInfo/MemoryInfoModel.cpp
//
// Created by klapeto on 21/3/2023.
//

#include "MemoryInfoModel.hpp"

namespace Elpida::Application
{
	MemoryInfoModel::MemoryInfoModel(size_t totalSize, size_t pageSize)
			: _totalSize(totalSize), _pageSize(pageSize)
	{
	}

	std::size_t MemoryInfoModel::GetTotalSize() const
	{
		return _totalSize;
	}

	std::size_t MemoryInfoModel::GetPageSize() const
	{
		return _pageSize;
	}
} // Application

File: apps/Mobile/Elpida.Mobile/Elpida.Mobile/Models/MemoryInfoModel.cs
namespace Elpida.Mobile.Models
{
	public class MemoryInfoModel
	{
		public ulong TotalSize { get; }
		public ulong PageSize { get; }

		public MemoryInfoModel(ulong totalSize, ulong pageSize)
		{
			TotalSize = totalSize;
			PageSize = pageSize;
		}
	}
}